models/payment/src/payment/payment-draft/presets/sample-data-fashion/payment-aus1.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  LocalizedString,
+  KeyReference,
+  Money,
+} from '@commercetools-test-data/commons';
+import {
+  CustomerDraft,
+  type TCustomerDraft,
+} from '@commercetools-test-data/customer';
+import * as PaymentDraft from '../..';
+import * as PaymentMethodInfo from '../../../../payment-method-info';
+import { PaymentStatusDraft } from '../../../../payment-status';
+import { TransactionDraft } from '../../../../transaction';
+import { TPaymentDraftBuilder } from '../../../types';
+
+const customerAUS = CustomerDraft.presets.sampleDataFashion
+  .sampleUsa()
+  .build<TCustomerDraft>();
+
+const paymentAUS1 = (): TPaymentDraftBuilder =>
+  PaymentDraft.presets
+    .empty()
+    .customer(KeyReference.presets.customer().key(customerAUS.key!))
+    .amountPlanned(Money.random().centAmount(4075).currencyCode('AUD'))
+    .paymentMethodInfo(
+      PaymentMethodInfo.random()
```

Comment:
Should this be the empty preset?
```suggestion
      PaymentMethodInfo.empty()
```